store/tikv/kv.go

```diff
@@ -82,10 +82,10 @@ type KVStore struct {
 	spMutex   sync.RWMutex  // this is used to update safePoint and spTime
 	closed    chan struct{} // this is used to nofity when the store is closed
 
-	resolveTSMu struct {
-		sync.RWMutex
-		resolveTS map[uint64]uint64 // storeID -> resolveTS
-	}
+	// storeID -> safeTS, stored as map[uint64]uint64
+	// safeTS here will be used during the Stale Read process,
+	// it indicates the safe timestamp point that can be used to read some old data with consistency ensuring.
```

Comment:
It may not be "old data" if there are no updates after the safe timestamp, we can emphasize the safe timestamp is usually stale though

Reply:
Addressed. PTAL again.